bgpd/bgp_evpn.c

```diff
@@ -5698,8 +5701,8 @@ int bgp_evpn_local_vni_del(struct bgp *bgp, vni_t vni)
 int bgp_evpn_local_vni_add(struct bgp *bgp, vni_t vni,
 			   struct in_addr originator_ip,
 			   vrf_id_t tenant_vrf_id,
-			   struct in_addr mcast_grp)
-
+			   struct in_addr mcast_grp,
+			   ifindex_t svi_ifindex)
```

Comment:
Unfortunately this will need to be re-orged again when the vlan-aware-bundle changes come in :(